bin/post_compile

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
```

Comment:
Only heroku seems to ever call `bin/post_compile` and I've added a check for the `DYNO` env var that it sets